scripts/tdnf-automatic

```diff
@@ -0,0 +1,566 @@
+  if [ "${update_type}" != "all" -a "${update_type}" != "security" ]; then
+    EchoErr "Unknown update type: ${update_type}"
+    return 1
+  fi
+
+  local emitter=0
+  local emit_file="$(CfgDataGet "emitter|emit_to_file")"
+  local emit_to_stdio="$(StrToBool "$(CfgDataGet "emitter|emit_to_stdio")")"
+
+  EchoDbg "[${FUNCNAME}:${LINENO}] Update type: '${update_type}' emit_to_stdio: '${emit_to_stdio}' emit_file: '${emit_file}'"
+
+  if [ "${emit_to_stdio}" = "1" -o -n "${emit_file}" ]; then
+    emitter=1
+    local sys_name="$(CfgDataGet "emitter|system_name")"
+    if [ -z "${sys_name}" ]; then
+      sys_name="$(hostname)"
+    fi
+
+    local msg="$(tdnf -c "${TdnfConf}" check-update)"
+    if [ "$?" != "0" ]; then
+      EchoErr "tdnf check-update failed..."
+      exit 1
+    fi
+    if [ -z "${msg}" ]; then
+      msg="\nNo updates available on ${sys_name}. System upto date...\n"
+      HandleEmitter "${msg}" "${emit_file}" "${emit_to_stdio}"
+      return 0
+    fi
+
+    if [ "${update_type}" = "all" ]; then
+      local f="$(mktemp)"
+      tdnf -c "${TdnfConf}" upgrade --assumeno 2>&1 | grep -Ev "Total|aborted|Upgrading" > "${f}"
+
+      local pkg=
+      local ver=
+      local tmp=
+      while IFS=' ' read pkg tmp ver tmp tmp tmp; do
+        if [ -n "${pkg}" -a -n "${ver}" ]; then
+          available_updates+="${pkg}-${ver}\n"
+        fi
+      done < "${f}"
+      rm -f "${f}"
+    else
+      available_updates="$(tdnf -c "${TdnfConf}" updateinfo info)"
+      if [ "$?" != "0" ]; then
+        EchoErr "tdnf updateinfo info failed..."
+        exit 1
+      fi
+    fi
+  fi
+
+  if [ "${install}" = 1 ]; then
+    [ "${update_type}" = "all" ] && tdnf -c "${TdnfConf}" upgrade -y || tdnf -c "${TdnfConf}" --security update -y
+    if [ "$?" != "0" ]; then
+      EchoErr "tdnf upgrade | tdnf --security update failed..."
+      exit 1
+    fi
+    if [ "${emitter}" = "1" ]; then
+      msg="\nThe following updates are applied on - ${sys_name}:\n${available_updates}\n"
+      msg+="\n--- Updates completed at $(date) ---\n"
+    fi
+  elif [ "${notify}" = 1 -a "${emitter}" = "1" ]; then
+    msg="\nThe following updates are available on - ${sys_name}:\n${available_updates}\n"
+  fi
+
+  if [ "${emitter}" = "1" ]; then
+    HandleEmitter "${msg}" "${emit_file}" "${emit_to_stdio}"
+  fi
+
+  return 0
+}
+
+ExitHandler()
+{
+  local exit_status="$?"
+  local msg=
+
+  msg="tdnf-automatic - completed with exit status ${exit_status} at $(date)...\n"
+  [ "${exit_status}" = "0" ] && echo -e "${msg}" || EchoErr "${msg}"
+
+  exit "${exit_status}"
+}
+
+trap ExitHandler EXIT
+
+Main()
+{
+  local timer=0
+  local notify=0
+  local install=0
+  local auto_conf=
+
+  EchoDbg "\n*** ${FUNCNAME} ***\n"
+
+  echo -e "\ntdnf-automatic - started at $(date)...\n"
+
+  EchoDbg "[${FUNCNAME}:${LINENO}] Parsing arg(s): '$@'"
+  # Parse arguements
+  while [[ $# -gt 0 ]]; do
+    key="$1"
+
+    case ${key} in
+      -c|--auto_conf)
```

Comment:
--auto_conf option supported? I see only -c and --conf options from help.

Reply:
Typo, thanks for finding this one :) I used '-c' all the time while testing